CHANGES.md

```diff
@@ -1,3 +1,6 @@
+# 0.200.9 (2025-07-22)
+- [UPGRADED] Upgraded com.ibm.cloud:cloudant from 0.9.3 to 0.10.6.
```

Comment:
```suggestion
- [UPGRADED] Upgraded com.ibm.cloud:cloudant from 0.10.0 to 0.10.6.
```

Reply:
haha, thanks!

Reply:
fixed in [53a314d](https://github.com/IBM/cloudant-kafka-connector/pull/240/commits/53a314d04f758c8bd9fec2d2bc98525c82a64b66) (also the description of the PR.